.circleci/config.yml

```diff
@@ -1,9 +1,10 @@
 version: 2
 jobs:
   build:
-    working_directory: ~/code
+    working_directory: ~/flyve
```

Comment:
Keep in mind, the registered name is: Flyve MDM
You can use: flyve-mdm or flyve_mdm as slug.